js/debug.js

```diff
@@ -0,0 +1,122 @@
+//Global imports and Declarations
+
+import { SELF_URL } from './constants.js';
+let profilePic = document.querySelector('.img-holder');
+let username = document.querySelector('.username');
+let fullname = document.querySelector('.fullname');
+let rdsId = document.querySelector('.id');
+let discordJoinedAt = document.querySelector('.discordJoinedAt');
+let github_display_name = document.querySelector('.github_display_name');
+let incompleteUserDetails = document.querySelector('.incompleteUserDetails');
+let all_roles = document.querySelector('.all-roles');
+let twitter_id = document.querySelector('.twitter_id');
+let github_id = document.querySelector('.github_id');
+let linkedin_id = document.querySelector('.linkedin_id');
+let discordId = document.querySelector('.discordId');
+let toggleButton = document.querySelector('.toggle-button');
+let toggle = document.querySelector('.toggle');
+let indicator = document.querySelector('.indicator');
+let msg = document.querySelector('.suMsg');
+let popUp = document.querySelector('.pop-up');
+let website = document.querySelector('.website');
+
+let feilds = [
+  profilePic,
+  username,
+  fullname,
+  rdsId,
+  discordJoinedAt,
+  github_display_name,
+  incompleteUserDetails,
+  all_roles,
+  twitter_id,
+  github_id,
+  linkedin_id,
+  discordId,
+  indicator,
+  website,
+];
+
+//This function updates all the data feilds based on the the response we got from the api call,
+//such that feild exits of
+const handleViewUserDetails = async (result) => {
+  feilds.forEach((feild) => {
+    switch (feild.className) {
+      case 'fullname':
+        feild.innerText = (result.first_name + result.last_name).toUpperCase();
```

Comment:
I think it's okay to capitalize the first letter, not the full name if it is not recommended